packages/graphql/src/connections/Knex/KnexRaw/KnexRawSchema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Lowdefy Request Schema - KnexRaw",
+  "type": "object",
+  "required": ["query"],
+  "properties": {
+    "query": {
+      "type": "string",
+      "description": "SQL query string.",
+      "errorMessage": {
+        "type": "KnexRaw request property \"query\" should be a string."
```

Comment:
Again not sure if table name should be here, or must you say `FROM "tablename"` ... ?